src/autologin/AutoLogin.cpp

```diff
@@ -14,7 +14,9 @@
 
 // AutoLogin code shared between MacroQuest.exe and MQ2AutoLogin.dll
 
-#include "AutoLoginShared.h"
+#include "AutoLogin.h"
```

Comment:
src/autologin is easy to confuse with src/plugins/autologin

please give this folder a name identifying that it is the shared component.

Reply:
Changed the name to login